### PR TITLE
eurostat mothers fertility rate and age

### PR DESCRIPTION
**Pull request by eftekhari-mhs**

### CODE REVIEW

File: scripts/eurostat/regional_statistics_by_nuts/fertility_rate_mother_age/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+### Download URL
+
+[TSV] file is available for [download](https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/BulkDownloadListing?file=data/demo_r_find3.tsv.gz).
+
+[Data explorer](https://appsso.eurostat.ec.europa.eu/nui/show.do?dataset=demo_r_find3&lang=en)
+
+### Overview
+
+This dataset has the fertility rate, median and mean age of mother at childbirth for European Union (EU) countries to their NUTS3 geos over [2013-2018]. Codes used in the dataset:
```
Comment (jeffreyoldham) [inline scripts/eurostat/regional_statistics_by_nuts/fertility_rate_mother_age/README.md:20]:
```suggestion
This dataset has the fertility rate, median and mean age of mother at childbirth for European Union (EU) countries specified by their NUTS3 geos over [2013-2018]. Codes used in the dataset: 

```

File: scripts/eurostat/regional_statistics_by_nuts/fertility_rate_mother_age/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+1. Date: Years from 2013 to 2018 with some holes in time series marked with note "b" in the original dataset
+2. indic_de,unit,geo: e.g. AGEMOTH,YR,AL	 (Mean age of women at childbirth, Years, nuts/AL)
+
+indic_de,unit,geo is further broken down into:
+1. indic_de: 'TOTFERRT', 'AGEMOTH', 'MEDAGEMOTH'
+2. unit: 'YR' (age), 'NR' (fertility rate)
+3. geo: NUTS3 codes for regions of Europe
+
+The cleaned csv is broken up into the following major families of variables:
+1. Date: Subset of years from 2000 to 2019 with values in the original dataset
```
Comment (jeffreyoldham) [inline scripts/eurostat/regional_statistics_by_nuts/fertility_rate_mother_age/README.md:37]: I am confused how the original dataset has data from 2013 to 2018 and the final CSV has data from 2000 to 2019.

Also, does this mean the dataset is no longer being updated?

File: scripts/eurostat/regional_statistics_by_nuts/fertility_rate_mother_age/fertility_rate_preprocess_gen_tmcf.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+import pandas as pd
+import io
+import csv
+
+_DATA_URL = "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/BulkDownloadListing?file=data/demo_r_find3.tsv.gz"
+_CLEANED_CSV = "./Eurostats_NUTS3_FRate_Age.csv"
+_TMCF = "./Eurostats_NUTS3_FRate_Age.tmcf"
+
+_OUTPUT_CLOUMNS = [
```
Comment (jeffreyoldham) [inline scripts/eurostat/regional_statistics_by_nuts/fertility_rate_mother_age/fertility_rate_preprocess_gen_tmcf.py:23]:
```suggestion
_OUTPUT_COLUMNS = [
```

Please also correct all uses.

**Review by jeffreyoldham — COMMENTED** [review]
Thanks.

File: scripts/eurostat/regional_statistics_by_nuts/fertility_rate_mother_age/README.md (hunk 2 above)
Comment (eftekhari-mhs) [reply scripts/eurostat/regional_statistics_by_nuts/fertility_rate_mother_age/README.md:37]: This is a typo, it should be 2013-2018. Thanks for catching it!

**Review by jeffreyoldham — APPROVED** [review]
Thanks.

**Pull request merged**